includes/sanitizers/class-amp-object-sanitizer.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Class AMP_Object_Sanitizer
+ *
+ * @package AMP
+ */
+
+use AmpProject\Attribute;
+use AmpProject\Layout;
+
+/**
+ * Class AMP_Object_Sanitizer
+ *
+ * Sanitizes `<object>` embeds.
+ *
+ * @since 2.1
+ * @internal
+ */
+class AMP_Object_Sanitizer extends AMP_Base_Sanitizer {
+
+	/**
+	 * Default PDF embed height.
+	 *
+	 * @var int
+	 */
+	const DEFAULT_PDF_EMBED_HEIGHT = 600;
+
+	/**
+	 * Sanitize `object` elements from the HTML contained in this instance's Dom\Document.
+	 */
+	public function sanitize() {
+		$elements = $this->dom->getElementsByTagName( 'object' );
+
+		if ( 0 === $elements->length ) {
+			return;
+		}
+
+		/** @var DOMElement $element */
+		foreach ( $elements as $element ) {
+			if ( $element->getAttribute( Attribute::TYPE ) === 'application/pdf' ) {
```

Comment:
Requires `composer update` for https://github.com/ampproject/amp-toolbox-php/pull/169/files?diff=unified&w=1
```suggestion
			if ( $element->getAttribute( Attribute::TYPE ) === Attribute::TYPE_PDF ) {
```